### ISSUE TITLE
Missing country flag emojis

### ISSUE THREAD

**Issue by Benjin** [microsoft/fluentui-emoji#25]
Hi!  This emoji set appears to be incomplete, as it's missing the Unicode standard country emoji flags.  I, and many others, would love to be able to represent our nationalities, identities, and travel by using these flags in the wonderful Fluent style.

**Comment by justsomederpystuff** [comment]
I believe Regional Indicators are missing as a whole, even if we can't get flags for now these would be great to have.

**Comment by Benjin** [comment]
> I believe Regional Indicators are missing as a whole, even if we can't get flags for now these would be great to have.

While important to include in a font, would those be useful to release graphics of here?  I'm specifically looking at flags, something virtually every other mature emoji font has.

**Comment by TomasHubelbauer** [comment]
See #28 for why flags are not included. From https://medium.com/microsoft-design/designing-in-the-open-source-5c62be73a599 the Fine Print section:

> ...due to (legal speak, legal speak, legal speak), we can’t open-source trademarks. For the same reason, our country flags, video game, and technologist emoji also won’t be available.

**Comment by JeffJacobson** [comment]
> See #28 for why flags are not included. From https://medium.com/microsoft-design/designing-in-the-open-source-5c62be73a599 the Fine Print section:
> 
> > ...due to (legal speak, legal speak, legal speak), we can’t open-source trademarks. For the same reason, our country flags, video game, and technologist emoji also won’t be available.

Is this why none of the national flag emojis show up in the :window: + `.` dialog? 

Why is **[crossed flags](https://github.com/microsoft/fluentui-emoji/tree/558ab792aec59eb639671d41c5666ef12f6d0d80/assets/Crossed%20flags)**, which has two Japanese flags (🎌), allowed if country flags are supposed to be omitted?

**Comment by Benjin** [comment]
> See #28 for why flags are not included. From https://medium.com/microsoft-design/designing-in-the-open-source-5c62be73a599 the Fine Print section:
> 
> > ...due to (legal speak, legal speak, legal speak), we can’t open-source trademarks. For the same reason, our country flags, video game, and technologist emoji also won’t be available.

Yup, drawing attention that reason is precisely why I opened this issue.  Every other major company with their own emoji set has managed to navigate the legal complexities of including country flag emoji on their platforms.  Microsoft's not incapable of doing the same, just unwilling.

Also, that post says "our country flags [...] emoji also won't be available", which isn't fully truthful: they're not just unavailable in this open-sourced repo, they don't exist... not in Teams, not in Windows, not in any Microsoft product.

**Comment by aetinx** [comment]
It's sad that there are no country flags, but making them should be easy.

<img width="411" alt="image" src="https://user-images.githubusercontent.com/65425469/184361362-3b925a94-816d-48cb-a170-8e3f9af1ce7e.png">

**Comment by JeffJacobson** [comment]
Maybe there could be a script that gets the flags from another repo (e.g., [emcrisostomo/flags]). So the flags would not be stored in this repo, but users would run the script to download the flags from the external source.

Alternatively, if Microsoft doesn't want the above implemented in this repo, others could create their own repos that include both this one and [another one with national flags][emcrisostomo/flags] as submodules, then run a script in their own repos to combine them.

[emcrisostomo/flags]:https://github.com/emcrisostomo/flags

**Comment by nilsbondars** [comment]
Hello, guys! I'm making my own country flag set in the Fluent style, right now I'm 66/261 deep. I'm making them manually xD But I think I'm gonna be finished in a week max, so hmu, if you'll need them.

**Comment by aetinx** [comment]
> Hello, guys! I'm making my own country flag set in the Fluent style, right now I'm 66/261 deep. I'm making them manually xD But I think I'm gonna be finished in a week max, so hmu, if you'll need them.

What a legend!

**Comment by JeffJacobson** [comment]
> > Hello, guys! I'm making my own country flag set in the Fluent style, right now I'm 66/261 deep. I'm making them manually xD But I think I'm gonna be finished in a week max, so hmu, if you'll need them.
> 
> What a legend!

Where is this quote from?

**Comment by aetinx** [comment]
> > > Hello, guys! I'm making my own country flag set in the Fluent style, right now I'm 66/261 deep. I'm making them manually xD But I think I'm gonna be finished in a week max, so hmu, if you'll need them.
> > 
> > 
> > What a legend!
> 
> Where is this quote from?

Maybe the comment was deleted (or hidden)?

**Comment by laGameTV** [comment]
> > Hello, guys! I'm making my own country flag set in the Fluent style, right now I'm 66/261 deep. I'm making them manually xD But I think I'm gonna be finished in a week max, so hmu, if you'll need them.
> 
> What a legend!

Did anybody find a repo like this?

**Comment by LeftTwixWand** [comment]
What a joke......
